pkgs/development/python-modules/sjcl/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "sjcl";
+  version = "0.2.1";
+
+  format = "pyproject";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3761ba01f9f049eaa2f67516054b7bfb2e27fe8bc7c9c630da7168fba58f0c57";
+  };
+
+  propagatedBuildInputs = [ pycrypto ];
+
+  doCheck = false;
```

Comment:
We enable tests for python packages since the ecosystem is quite fragile and we need to be able to notice breakages early or else.

The pypi sdist might not include them, but this should be verified. If that is the cae we might need to pull the source from GitHub using `fetchFromGitHub`

The check phase should look like so:

```nix
  checkPhase = ''
    runHook preCheck
    ${python.interpreter} -m unittest discover
    runHook postCheck
  '';
```

This requires `python` to be added to the function argument. Also note that we call `runHook` since the default `checkPhase` takes care of that, but we overwrite it, so we need that so `preCheck` and `postCheck` still work, if needed.

Reply:
added with 71564e2a8a06ac1b6840608b3753f806bb730c93